src/index.html

```diff
@@ -4,9 +4,474 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header body__header">
+      <div class="header__container">
+        <a href="index.html">
+          <img
+            class="header__logo"
+            src="./images/logo.png" alt="logo"
+            >
+        </a>
+
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li>
+              <a class="is-active nav__link" href="#">Apple</a>
+            </li>
+
+            <li>
+              <a href="#samsung" class="nav__link">Samsung</a>
+            </li>
+
+            <li>
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+
+            <li>
+              <a
+                data-qa="nav-hover"
+                href="#"
+                class="nav__link"
+                >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li>
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+
+            <li>
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+
+            <li>
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+
+            <li>
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="body__main main">
+      <ul class="product-cards main__product-cards">
+        <li class="product-cards__item">
+          <div class="card product-cards__card" data-qa="card">
```

Comment:
A note for the future: Selectors are good, but do not overcomplicate your code. If you didn't use the unordered list, you would have a block "main" as a wrapper and blocks "card" as their content. Adding a ul made you introduce and new block that will, most likely see little use outside of this construction. Div selectors are frowned upon, but they still have their place in our code.